src/lib.rs

```diff
@@ -168,6 +168,198 @@ pub fn builder() -> Builder {
 }
 
 impl Builder {
+    /// Configure and generate Rust bindings for a C/C++ header.
+    ///
+    /// This is the main entry point to the library.
```

Comment:
This is not supposed to be the main entry point to the library. This is just a utility for debugging what was originally `build.rs` usage with command line flags instead.

For example, Stylo uses the `Builder` API in `build.rs`, but it is easiest to run `creduce` with the command line API. So when I am debugging a Stylo + bindgen bug, the first thing I do is try and convert the `Builder` API usage into equivalent command line flags and arguments so I can use `creduce` on it.

Nothing here should be mutating a `Builder`, and it doesn't seem like the `builder_ref` is mutated so it shouldn't be necessary to take an `&mut` reference.

Here is the type signature skeleton I would expect:

```
impl Builder {
    /// For debugging purposes, dump the set of command line flags that is
    /// equivalent to this builder invocation.
    pub fn dump_command_line_flags(&self) -> Vec<String> {
        // ...
    }
}